pkg/controller/storagecluster/placement_test.go

```diff
@@ -94,7 +94,49 @@ func TestGetPlacement(t *testing.T) {
 	}
 	assert.Equal(t, mockPlacements["all"], getPlacement(sc, "all"))
 
-	// Case 4: The LabelSelector modifies the configured Placements correctly
+	// Case 4: LabelSelector modifies an empty NodeAffinity
+	sc = &ocsv1.StorageCluster{}
+	mockStorageCluster.DeepCopyInto(sc)
+	sc.Spec.Placement = map[rookv1.KeyType]rookv1.Placement{
+		"all": rookv1.Placement{
+			Tolerations: []corev1.Toleration{
+				corev1.Toleration{
+					Key:      defaults.NodeTolerationKey,
+					Operator: corev1.TolerationOpEqual,
+					Value:    "true",
+					Effect:   corev1.TaintEffectNoSchedule,
+				},
+			},
```

Comment:
```suggestion
			Tolerations: defaults.DaemonPlacements["all"].Tolerations,
```

Isn't this the same?

(this is in the commit `Update TestGetPlacement() for proper LabelSelector behavior `)